iron-overlay-manager.html

```diff
@@ -43,10 +43,10 @@
     // NOTE: Use useCapture=true to avoid accidentally prevention of the closing
     // of an overlay via event.stopPropagation(). The only way to prevent
     // closing of an overlay should be through its APIs.
-    Polymer.Gestures.add(document, 'tap', null);
-    document.addEventListener('tap', this._onCaptureClick.bind(this), true);
-    document.addEventListener('focus', this._onCaptureFocus.bind(this), true);
-    document.addEventListener('keydown', this._onCaptureKeyDown.bind(this), true);
+    Polymer.Gestures.add(document.documentElement, 'tap', null);
+    document.documentElement.addEventListener('tap', this._onCaptureClick.bind(this), true);
```

Comment:
I think this is the only line that needs to be changed in order to fix the issue you're facing